diff --git a/setup.py b/setup.py
index 11f192e0..8a68a46c 100644
--- a/setup.py
+++ b/setup.py
@@ -79,7 +79,7 @@ setup(name='dace',
       include_package_data=True,
       install_requires=[
           'numpy', 'networkx >= 2.5', 'astunparse', 'sympy<1.9', 'pyyaml', 'ply',
-          'websockets', 'requests', 'flask', 'scikit-build', 'cmake', 'aenum',
+          'websockets', 'requests', 'flask', 'scikit-build', 'aenum',
           'dataclasses; python_version < "3.7"', 'dill', 'pyreadline;platform_system=="Windows"'
       ],
       extras_require={
